apiFunctions.php

```diff
@@ -451,7 +451,7 @@ function expand_doi_with_dx($template, $doi) {
      if (!$doi) return FALSE;
      $ch = curl_init();
      curl_setopt($ch, CURLOPT_URL,'https://doi.org/' . $doi);
-     curl_setopt($ch, CURLOPT_HTTPHEADER, array("Accept: application/x-research-info-systems"));
+     curl_setopt($ch, CURLOPT_HTTPHEADER, array("Accept: 	application/vnd.citationstyles.csl+json"));
```

Comment:
Is the large whitespace before `application` intentional?
```suggestion
     curl_setopt($ch, CURLOPT_HTTPHEADER, array("Accept: application/vnd.citationstyles.csl+json"));
```